applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_fast_pair.conf

```diff
@@ -97,6 +106,25 @@ CONFIG_PWM=y
 CONFIG_LED=y
 CONFIG_LED_PWM=y
 
+CONFIG_NET_BUF=y
+CONFIG_ZCBOR=y
+CONFIG_CRC=y
+CONFIG_MCUMGR=y
+CONFIG_MCUMGR_TRANSPORT_BT=y
+CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN=n
+
+CONFIG_STREAM_FLASH=y
+CONFIG_IMG_MANAGER=y
+CONFIG_MCUBOOT_IMG_MANAGER=y
+CONFIG_MCUMGR_GRP_IMG=y
+CONFIG_MCUMGR_GRP_OS=y
+
+CONFIG_MCUMGR_MGMT_NOTIFICATION_HOOKS=y
+CONFIG_MCUMGR_GRP_IMG_UPLOAD_CHECK_HOOK=y
```

Comment:
Some additional hooks may be required. Consider either waiting with this PR for my change or removing the MCUmgr scope from this PR

Reply:
I use the PR to test MCUmgr with XIP locally so I would prefer to leave it as it is. Later on I can switch to your new module's implementation